thesis-de/content-de.tex

```diff
@@ -1888,16 +1888,104 @@ \section{Userstories}
 Tickets arbeiten können.
 \tm%
 
-\subsection{Userstory 1}
+\subsection{SSD Userstory}
+\textbf{Name der Story:} Anzeige der SSD's, welche einen Hardwaredeffekt haben könnten.
+\textbf{Beschreibung:} In der heutigen Zeit, werden vermehrt langsame Hard Disk Drive's
+durch schnellere Solid State Disk's ausgetauscht. Gerade im Serverbetrieb, sorgt
+dies für einen schnelleren Datenzugriff. SSDs haben jedoch, wie auch HDDs einen
+Lebenszyklus, der bei SSDs schneller erreicht werden kann, als bei 
+handelsüblichen HDDs. Der Administrator soll hier darüber Benachrichtigt werden,
+sobald von einer SSD ein kritischer Schreib- oder Lesezyklus erreicht worden
+ist, um die SSD, bereits vor dem Ausfall austauschen zu können. Ebenfalls soll
+hier auch eine Statistik ausgegeben werden, welche SSD von welchem Hersteller in
+der Wahrscheinlichkeit am häufigsten ausfallen kann.
+\textbf{Akzeptanzkriterien:}
 
-\subsection{Userstory 2}
+\begin{outline}
+  \1 Die Ausgabe und Visualisierung in einem Balken- oder Graphendiagramm
+  für jede einzelne Instanz
+  \1 Auflistung einer Statistik in Form eines Balkendiagramms, über die
+  Hersteller, welche am wahrscheinlich häufigsten in den Instanzen ausfallen
+  \1 Eine Benachrichtigung, ob per Email oder andere Kommunikationswege,
+  wenn ein kritischer Schreib- oder Lesezyklus erreicht worden ist
+\end{outline}
+\mr%
 
-\subsection{Frontend}
-Name der Story: Visualisierung der erhaltenen Daten zur weiteren Verwendung
-Beschreibung: Dem Administrator oder dem User soll es ermöglicht werden, eine
+\subsection{CPU Userstory}
+\textbf{Name der Story:} Auswertung der CPU Auslastung jeder Instanz, um Engpässe zu
+erkennen.
+\textbf{Beschreibung:}\\ Die CPUs der heutigen Zeit werden in der Taktrate und
+Verarbeitungsgeschwindigkeit der Prozesse immer schneller und besser. Bei
+virtuellen Maschinen, werden aus einem physischen CPU Kern des Host Systems
+mehrere virtuelle CPU Kerne emuliert und einer Instanz zugewiesen. So kann eine
+VM 2 virtuelle CPU Kerne mit z.B. 2,5GHz Takrate zugewiesen haben. Auf jeder
+Instanz laufen andere Dienste, weshalb jede VM unterschiedliche CPUs zugeteilt
+haben kann. Hier kann es vorkommen, dass eine VM mehr CPU Leistung benötigt, als
+ihr selber zugewiesen ist. Um dies frühstmöglich zu erkennen, benötigt der
+Administrator eine detailierte Übersicht in Form eines Graphen, wieviel CPU
+Leistung von einer virtuellen Instanz verwendet wird und auf dem System
+verfügbar ist.
+\textbf{Akzeptanzkriterien:}
+
+\begin{outline}
+  \1 Visualisierung der Auslastung und Verfügbarkeit der CPU Leistung für jede
+  einzelne Instanz
+  \1 Alle Werte der CPU, wie \gls{soft-IRQ}, wait oder Interrupt müssen visualisiert
```

Comment:
mhh oke ja habe ich notiert und werde ich gleich in nem extra PR ins Glossar packen und die stellen dann noch anpassen